src/specific/s_input.c

```diff
@@ -590,27 +1020,127 @@ INPUT_SCANCODE S_Input_GetAssignedScancode(
     return m_Layout[layout_num][role];
 }
 
+int16_t S_Input_GetUniqueBind(INPUT_LAYOUT layout_num, INPUT_ROLE role)
+{
+    CONTROLLER_MAP assigned = m_ControllerLayout[layout_num][role];
+    if (assigned.type == BUTTON) {
+        return m_ControllerLayout[layout_num][role].bind.btn;
+    } else {
```

Comment:
Unnecessary `else`, can either be merged to `else if` down below or removed altogether